UnityProject/WhileYoureHere/Assets/Scripts/radio interaction/RadioTracks.cs

```diff
@@ -0,0 +1,11 @@
+using UnityEngine;
+
+namespace radio_interaction
+{
+    [CreateAssetMenu(fileName = "RadioTracks", menuName = "Scriptable Objects/RadioTracks")]
```

Comment:
```suggestion
    [CreateAssetMenu(fileName = "RadioTracks", menuName = "ScriptableObjects/RadioTracks")]
```
Remove the space here as well, same as the other one, this creates an unwanted entry in the asset menu.